tests/installation/setup_libyui.pm

```diff
@@ -39,29 +39,31 @@ sub run {
                 return $+{ip} if ($ip =~ $ip_regexp);
         }, timeout => $boot_timeout, interval => 30);
         set_var('YUI_SERVER', $ip);
+        select_console('sut', await_console => 0);
     } elsif (check_var('BACKEND', 's390x')) {
         select_console('install-shell');
         my $ip = YuiRestClient::Wait::wait_until(object => sub {
                 my $ip = script_output('ip -o -4 addr list | sed -n 2p | awk \'{print $4}\' | cut -d/ -f1', proceed_on_failure => 1);
                 return $+{ip} if ($ip =~ $ip_regexp);
         });
         set_var('YUI_SERVER', $ip);
+        select_console('installation');
     } elsif (check_var('VIRSH_VMM_FAMILY', 'xen')) {
+        # For xen, when attempting to switch console while the installation loader is not finished, we end up with test failure.
         assert_screen "inst-betawarning", 500;
         select_console('root-console');
         my $ip = YuiRestClient::Wait::wait_until(object => sub {
                 my $ip = script_output('ip -o -4 addr list | sed -n 2p | awk \'{print $4}\' | cut -d/ -f1', proceed_on_failure => 1);
                 return $+{ip} if ($ip =~ $ip_regexp);
         });
         set_var('YUI_SERVER', $ip);
+        select_console('installation');
     } elsif (is_ssh_installation) {
         my $cmd = (is_s390x && is_svirt) ? "TERM=linux " : "";
         $cmd .= YuiRestClient::get_yui_params_string() . " yast.ssh";
         enter_cmd($cmd);
```

Comment:
do we need here 'select_console' ?

Reply:
Should not be necessary, as we are already on ssh. For the cases we have now - it will work.
What I've noticed though is that we still have `            # On s390x zKVM we have to process startshell in bootloader
            YuiRestClient::setup_libyui() if YuiRestClient::is_libyui_rest_api;` in bootloader_zkvm =)